manager/client/main.go

```diff
@@ -87,7 +88,11 @@ func polling(client rpc_license.LicenseClient, username string, quantity int32,
 func main() {
 	var quantity int32 = 1
 	host, port := os.Args[1], os.Args[2]
-	vendor, feature := os.Args[3], os.Args[4]
+	timeout, err := strconv.Atoi(os.Args[3])
```

Comment:
if you do it like this, we will be forced to update enkit and the eda rules in lockstep, as the new client is backward incompatible with the previous one.

Add flag parsing, really. All you need is something like:

```
timeout = flag.Duration("timeout", 20 * time.Minute,  "How long to hold the license for at most")
```
and then add:
```
flag.Parse()
```
of course with an:
```
import "flag"
```
statement.